src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -16,6 +17,7 @@ public class UserPrefs implements ReadOnlyUserPrefs {
     private GuiSettings guiSettings = new GuiSettings();
     private Path addressBookFilePath = Paths.get("data" , "addressbook.json");
 
+    private Mod teaching;
```

Comment:
How does this field influence startup behavior at the moment?

Reply:
The title of the window changes (refer to screenshot). The list of TAs are filtered according to the default course automatically.